clang/lib/Sema/AnalysisBasedWarnings.cpp

```diff
@@ -2256,6 +2256,10 @@ class UnsafeBufferUsageReporter : public UnsafeBufferUsageHandler {
         Range = UO->getSubExpr()->getSourceRange();
         MsgParam = 1;
       }
+    } else if (const auto *CtorExpr = dyn_cast<CXXConstructExpr>(Operation)) {
+      S.Diag(CtorExpr->getLocation(),
+             diag::warn_unsafe_buffer_usage_in_container)
+          << 0;
```

Comment:
Assuming we remove the warning message parameter we should not pass the argument here.